Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -83,6 +83,18 @@ void ImageEditor::paint_event(GUI::PaintEvent& event)
         painter.draw_rect(enclosing_int_rect(image_rect_to_editor_rect(m_active_layer->relative_rect())).inflated(2, 2), Color::Black);
     }
 
+    for (auto& guide : m_guides) {
```

Comment:
In the commit message:

_The ImageEditor knows about it's Guides_
-> _The ImageEditor knows about its Guides_

(please do spellcheck them all :)